appinventor/components/src/edu/mit/appinventor/CloudDB.java

```diff
@@ -58,13 +74,14 @@
         "server to store and retrieve information.",
     category = ComponentCategory.EXTENSION,
     nonVisible = true,
-    iconName = "images/cloudDB.png")
+    iconName = "http://web.mit.edu/graeme/www/appinventor/cloudDB.png")
```

Comment:
Will this still be up after you graduate? Hopefully we'll migrate CloudDB to Experimental before then, but just in case